bindgen/src/output-directory.ts

```diff
@@ -93,9 +92,9 @@ export function createOutputDirectory(outputPath: string): OutputDirectory {
       }
     },
     format() {
-      for (const formatterName of getFormatterNames()) {
-        const relevantFiles = openFiles.filter((f) => f.formatters.includes(formatterName)).map((f) => f.resolvedPath);
-        format(formatterName, outputPath, relevantFiles);
+      for (const formatter of usedFormatters) {
```

Comment:
I was about to comment that because `usedFormatters` is a `Set` you don't have any guaranteed order. However it looks like you do, although you need to read [the spec](https://tc39.es/ecma262/multipage/keyed-collections.html#sec-createsetiterator) a bit carefully to see that. It is a bit weird that [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/@@iterator) doesn't say anything about the order at all.

Reply:
I filed https://github.com/mdn/content/pull/26727